packages/components/link/src/link.visualroute.js

```diff
@@ -35,8 +35,17 @@ export const component = () => (
     <Spec label="intlMessage">
       <Link to="/" intlMessage={intlMessage} />
     </Spec>
-    <Spec label="white">
-      <Link to="/" tone={'inverted'} />
-    </Spec>
+    <ThemeProvider theme={themes.darkTheme}>
+      <Spec label="tone - inverted - with underline">
```

Comment:
Underlines can be achieved in different ways. E.g. with border or natively to the font. As a result I think it's beneficial to test both in a VRT for an inverted option.